docs/modules/operation/pages/provisioning/auto-discovery.adoc

```diff
@@ -72,3 +72,103 @@ A definition without any detectors will fall back to ICMP ping for discovery.
 
 </discovery-configuration>
 ----
+
+[[ga-manual-discovery]]
+== Manual discovery configuration
+
+In addition to the configuration above, there are some optional settings you can configure via `$\{OPENNMS_HOME}/etc/discovery-configuration.xml`.
+
+.Discovery configuration parameters
+[options="header" cols="1,1,3"]
+|===
+| Name
+| Default
+| Description
+
+| packets-per-second
+| 1
+| Number of ICMP packets that will be generated each second.
+
+| initial-sleep-time
+| 30000 ms (30 seconds)
+| The time, in milliseconds, before the discovery process begins after OpenNMS has started.
+This delay allows {page-component-title} to fully start before generating new events.
+
+| restart-sleep-time
+| 86400000 ms (1 day)
+| Once the discovery process finishes, the time, in milliseconds, before it will start again.
+
+| timeout
+| 2000 ms
+| The amount of time, in milliseconds, that the discovery process will wait for a response from a given IP address before deciding that there is nothing there.
+
+| retries
+| 1
+| The number of attempts to query a given IP address before deciding that there is nothing there.
+
+| foreign-source
+| not set
+| Adds discovered nodes to the specified foreign source requisition.
+| location
```

Comment:
Is this a formatting issue?  No blank line between FS section and location section

Reply:
```suggestion

| location
```